source/tasks/CreateOctopusRelease/CreateOctopusReleaseV6/inputCommandBuilder.ts

```diff
@@ -64,6 +66,13 @@ export function createCommandFromInputs(logger: Logger, task: TaskWrapper): Crea
         GitCommit: task.getInput("GitCommit"),
     };
 
+    if (!command.ReleaseNotes) {
```

Comment:
In the [cli](https://github.com/OctopusDeploy/cli/blob/main/pkg/cmd/release/create/create.go#L200) we fail the command if both release notes and release notes file are specified, I wonder if we should do the same here to prevent customers hitting any unexpected behavior?

Reply:
Good point, can change this to have the same behaviour